app/login/components/profile-content-tabs.tsx

```diff
@@ -35,11 +35,13 @@ const useStyles = makeStyles<Theme>((theme) => ({
 }));
 
 type ProfileContentTabsProps = {
+  userId: number;
   userConfigs: ParsedConfig[];
 };
 
 export const ProfileContentTabs = (props: ProfileContentTabsProps) => {
-  const { userConfigs } = props;
+  const { userId, userConfigs: _userConfigs } = props;
+  const [userConfigs, setUserConfigs] = React.useState(_userConfigs);
```

Comment:
Not sure I understand the change 🤔 ?

Reply:
Ah, this commit was a preparation for a next one, probably they should be one – as we fetch the user configs on the server side, there was a need to `useState` so that when user removes a chart, we can simply handle the change on the frontend, without re-querying configs (+ to make the table re-render once it's removed).